routes/commentRouter.js

```diff
@@ -4,8 +4,14 @@ const commentController = require("../controllers/commentController");
 
 router.post("/createComment", commentController.createComment);
 
-router.get("/comment", commentController.getCommentsbySender);
+router.get("/comment", commentController.getCommentsBySender);
 
 router.get("/comment/:id", commentController.getCommentById);
 
+router.get('/post/:post_id', commentController.getCommentsByPost);
+
+router.put('/:id', commentController.updateComment);
+
+router.delete('/:id', commentController.deleteComment);
```

Comment:
change it to /comment/:id or change getCommentsBySender and getCommentById to / and /:id